lib/tasks/test_replication.rake

```diff
@@ -5,15 +5,26 @@ namespace :test do
   namespace :replication do
     desc "Setup environment for replication specs"
     task :setup => :initialize do
-      EvmTestSetupReplication.new.execute
+      EvmTestSetupReplication.new.execute_setup
+    end
+
+    desc "Teardown environment for replication specs"
+    task :teardown => :initialize do
+      EvmTestSetupReplication.new.execute_teardown
     end
   end
 
   desc "Run all replication specs"
-  RSpec::Core::RakeTask.new(:replication => :initialize) do |t|
+  RSpec::Core::RakeTask.new(:replication => [:initialize, :replication_util]) do |t|
```

Comment:
I can confirm that it even runs in a different process.
I tried out printing the pids in the different spec files and got the same one for all of the util specs and a new one for the replication spec.